airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -202,53 +202,39 @@ class EcsDeregisterTaskDefinitionOperator(EcsBaseOperator):
 
     :param task_definition: The family and revision (family:revision) or full Amazon Resource Name (ARN)
         of the task definition to deregister. If you use a family name, you must specify a revision.
-    :param wait_for_completion: If True, waits for creation of the cluster to complete. (default: True)
-    :param waiter_delay: The amount of time in seconds to wait between attempts,
-        if not set then the default waiter value will be used.
-    :param waiter_max_attempts: The maximum number of attempts to be made,
-        if not set then the default waiter value will be used.
     """
 
-    template_fields: Sequence[str] = ("task_definition", "wait_for_completion")
+    template_fields: Sequence[str] = "task_definition"
```

Comment:
This is a breaking change
Even if it doesn't mean anything if user has templating over this field and we remove it his code will break.
We should keep it until we remove the parameter

Reply:
is it going to break ? 🤔 
My thinking was just that the "right" value wouldn't be resolved, but that ain't a problem if we don't use that value.

Reply:
like, jinja templates will still be strings with "{{ whatever }}" and xcom values will be some weird types, but nothing breaking I'd say

Reply:
You are right. resolving